### DISCUSSION TITLE
Tiff layer mask

### DISCUSSION THREAD

**Discussion by andrey-hider** [ImageMagick/ImageMagick#4741] [General]
Is there any way to extract a Layer mask thumbnail out of tiff image layer using ImageMagick?
It would be interesting for me to export black and white mask from the layer "Red Layer with Mask inside Folder".

Here how it looks like in Abode Photoshop:
![image](https://user-images.githubusercontent.com/30869869/150537663-0216c0bc-6899-45ed-84ec-a0e3af15c731.png)

Here is the example of [tiff file](https://github.com/ImageMagick/ImageMagick/files/7914064/Alpha.Masks.Sample.with.3.Alphas.zip).

**Comment by fmw42** [comment]
Did you try

`magick image.tif image.png`

**Comment by andrey-hider** [comment]
@fmw42 
Thanks for your reply.
I've got 5 images as a result. Looks like it is one overall preview and 4 layer images.
Is there a way to get image layer details?
For example, if layer contains layer mask.
Command `identify image.tif` wasn't helpful.

**Reply by fmw42** [reply]
I do not know, but doubt it. This would have to be answered by one of the IM developers.
